docs/monitoring/flow.rst

```diff
@@ -0,0 +1,68 @@
+.. |MonitorZendesk| image:: ../imagesMonitorZendesk.jpg
+.. |MonitorTrigger| image:: ../images/MonitorTrigger.jpg
+.. |MonitorBuffered| image:: ../images/MonitorBuffered.jpg
+
+Pipeline Data Flow
+==================
+
+There are several aspect to monitoring the data pipelines within Fledge, the one we will discuss in this section is how to monitor the flow of data. This is about if data is flowing, not if the data that is flowing is good data, just because there is a flow of data says nothing about the quality of that data.
+
+The south services automatically monitor the flow rates out from the south service to the storage layer, the configuration of this monitoring is discussed in the section on tuning Fledge. This automatic monitoring uses statistical methods to detect changes in the flow rate and alerts the user via the user interface status bar alerts.
+
+The main mechanism for monitoring data flow is the statistics that are collected by all the services that implement a Fledge instance.
+
+South Service Data Flow
+-----------------------
+
+The south service maintains two distinct statistics, one for the total ingest count for the services and a set for the ingest counts for each asset ingested by the service. These statistics are collected when the data is sent from the south service to the storage service, they do therefore not include any readings that have been bought into the service by removed by any of the filters in the filter pipeline.
+
+The south service may also be configured to reduce the statistics it collects. Very heavily loaded south services, collecting large numbers of distinct assets are often configured to only collect total ingest statistics for the service as this can have a very large impact on the efficiency of the service.
+
+North Service Data Flow
+-----------------------
+
+The north services also collect statistics, in this case we can get the total number of readings sent to all north systems or the number sent to each north system. Unlike the south services both of these are always collected and so they can be utilised in all cases.
+
+Statistics Mechanism
+--------------------
+
+Statistics are gathered as pure counters that increment as data traversing the pipeline or whatever the statistic is monitoring, this is not particularly useful when trying to monitor rates. This is however another options, the statistics collector is a task that runs at periodic intervals and takes the raw statistics to create a statistics history table. The statistics collector is run by default every 15 seconds, although this interval can be altered via the configuration. The statistics table will therefore record the increment in each statistic every 15 seconds. This allows us to easily obtain a rate of change for any statistic gathered by the system. It is these statistics history entries that are shown by the Flog dashboard, thus it shows the number of readings ingested every 15 seconds by default.
```

Comment:
- "...that increment as data traversing..." -> "...that increment as data is traversing..."
- "...statistic is monitoring, this is not particularly..." Start new sentence at "This is not particularly..."
- "This is however another options" -> "There are however other options" (or, if there is only one: "There is another option.")
- Start new sentence at "The statistics collector is a task that runs..."
- "...by the Flog dashboard, thus it shows the number of readings..." -> "...by the Fledge dashboard: it shows the number of readings..."